Project_Starter_Files-Building_a_Metrics_Dashboard/reference-app/trial/app.py

```diff
@@ -1,73 +1,93 @@
-from flask import Flask, render_template, request, jsonify
+import logging
+import re
+import requests
 
+
+from flask import Flask, jsonify, render_template
+from flask_opentracing import FlaskTracing
 from jaeger_client import Config
 from jaeger_client.metrics.prometheus import PrometheusMetricsFactory
-from opentelemetry import trace
-from opentelemetry.exporter import jaeger
-from opentelemetry.sdk.trace import TracerProvider
-from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
```

Comment:
> Why some imports have been removed from Project_Starter_Files-Building_a_Metrics_Dashboard/reference-app/trial/app.py file?

@SudKul, 
- These imports aren't well explained as to why they are needed!
- They add complexity to the project and confusions for students as they aren't justified on the lessons
- Replaced all imports with `PrometheusMetrics` which does not or less the same